frontend/src/metabase/dashboard/components/ClickBehaviorSidebar/LinkOptions/LinkedEntityPicker.tsx

```diff
@@ -104,7 +168,7 @@ function TargetClickMappings({
   );
 }
 
-function QuestionDashboardPicker({
+function LinkedEntityPicker({
```

Comment:
Renaming since it's not really a `QuestionDashboardPicker` anymore and `QuestionDashboardPagePicker` is too much for me